cmd/selfupdate/selfupdate.go

```diff
@@ -221,7 +221,7 @@ func InstallUpdate(ctx context.Context, opt *Options) error {
 
 	err = replaceExecutable(targetFile, newFile, savedFile)
 	if err == nil {
-		fmt.Printf("Successfully updated rclone from version %s to version %s\n", oldVersion, newVersion)
+		fs.Infof(nil, "Successfully updated rclone from version %s to version %s", oldVersion, newVersion)
```

Comment:
Same comment as above:
Info is only shown when -v flag is present.
I think this message should be printed with fs.Logf (aka NOTICE) so it's visible by default. I'm just not sure that --quiet will hide it.

Reply:
Please use Logf here, too.

Reply:
this diff is outdated, please check the latest diff.🤣

Reply:
🧑‍🎓